cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/DRep.hs

```diff
@@ -117,22 +151,58 @@ hprop_golden_governance_drep_metadata_hash = propertyOnce . H.moduleWorkspace "t
     , "--out-file", outputDRepMetadataHash
     ]
 
-  H.diffFileVsGoldenFile goldenDRepMetadataHash outputDRepMetadataHash
+  H.diffFileVsGoldenFile outputDRepMetadataHash goldenDRepMetadataHash
+
+hprop_golden_governance_drep_registration_certificate_vkey_file :: Property
+hprop_golden_governance_drep_registration_certificate_vkey_file = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
+  drepVKeyFile <- noteInputFile "test/cardano-cli-golden/files/input/drep.vkey"
+  goldenFile <- noteInputFile "test/cardano-cli-golden/files/golden/governance/drep/drep_registration_certificate.json"
+
+  outFile <- H.noteTempFile tempDir "drep-reg-cert.txt"
+
+  void $ execCardanoCLI
+    [ "conway", "governance", "drep", "registration-certificate"
+    , "--drep-verification-key-file", drepVKeyFile
+    , "--key-reg-deposit-amt", "0"
+    , "--drep-metadata-url", "dummy-url"
+    , "--drep-metadata-hash", "52e69500a92d80f2126c836a4903dc582006709f004cf7a28ed648f732dff8d2"
+    , "--out-file", outFile
+    ]
+
+  H.diffFileVsGoldenFile outFile goldenFile
+
+hprop_golden_governance_drep_registration_certificate_id_hex :: Property
+hprop_golden_governance_drep_registration_certificate_id_hex = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
+  goldenFile <- noteInputFile "test/cardano-cli-golden/files/golden/governance/drep/drep_registration_certificate.json"
+  idGold <- H.readFile "test/cardano-cli-golden/files/golden/governance/drep/drep.id.hex"
+
+  outFile <- H.noteTempFile tempDir "drep-reg-cert.txt"
+
+  void $ execCardanoCLI
+    [ "conway", "governance", "drep", "registration-certificate"
+    , "--drep-key-hash", idGold
+    , "--key-reg-deposit-amt", "0"
+    , "--drep-metadata-url", "dummy-url"
+    , "--drep-metadata-hash", "52e69500a92d80f2126c836a4903dc582006709f004cf7a28ed648f732dff8d2"
+    , "--out-file", outFile
+    ]
+
+  H.diffFileVsGoldenFile outFile goldenFile
 
-hprop_golden_governance_drep_registration_certificate :: Property
-hprop_golden_governance_drep_registration_certificate = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
-  drepKeyFile <- noteInputFile "test/cardano-cli-golden/files/golden/governance/drep/drep.vkey"
+hprop_golden_governance_drep_registration_certificate_id_bech32 :: Property
+hprop_golden_governance_drep_registration_certificate_id_bech32 = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
   goldenFile <- noteInputFile "test/cardano-cli-golden/files/golden/governance/drep/drep_registration_certificate.json"
+  idGold <- H.readFile "test/cardano-cli-golden/files/golden/governance/drep/drep.id.bech32"
```

Comment:
It seems like this is an input file rather than a golden file.

If so, it should go under `files/input` rather than `files/golden`.

Ideally we want to be at the point where we can run:

```
rm -rf cardano-cli/test/cardano-cli-golden/files/golden
CREATE_GOLDEN_FILES=1 cabal test all --enable-tests
```

To regenerate all the golden files.